addons/web_editor/static/src/js/wysiwyg/widgets/link_dialog.js

```diff
@@ -77,6 +77,12 @@ export class LinkDialog extends Link {
         this.props.close();
     }
 
+    onKeyDownEnter(ev) {
```

Comment:
I suggest to rename this into `onUrlKeydown` because it is called on every `keydown` event in the url input,  and only in the url input and not only for Enter.

Reply:
I have made the changes.